### PR TITLE
docs: update landing page

### PR DESCRIPTION
**Pull request by zr-msft**
Signed-off-by: Zach Rhoads <zachary.rhoads@microsoft.com>

### CODE REVIEW

**Review by toddysm — CHANGES_REQUESTED** [review]

File: content/en/_index.html [hunk 1]
```diff
@@ -16,27 +16,30 @@
 {{< blocks/cover image_anchor="top" height="min" color="primary" >}}
 <img src="notary-horizontal-white.png" class="site-logo" />
 <h2 class="mt-4">
-  The Notary project comprises a server and a client for running and interacting
-with trusted collections.
+  Scenarios, specifications, and reference implementations for container secure supply chain.
```
[the remark this answers is not in the document]
Comment (FeynmanZhou) [reply content/en/_index.html:19]: Looks better. I think we might also include the keyword security here. How about "Extensible, standards-based specification and tooling for software security supply chains"?

File: content/en/_index.html [hunk 2]
```diff
@@ -16,27 +16,30 @@
...
-  <ul>
-    <li>Sign, store, and verify any OCI artifact, including container images</li>
-    <li>Signing and verifying uses public/private key verification</li>
-    <li>Signing does not change the <code>@digest</code> or <code>artifact:tag</code> reference</li>
-    <li>Signatures are portable across OCI-conformant registries</li>
-    <li>Support for multiple signatures</li>
-    <li>Designed to support multiple development flows</li>
-  </ul>
+<h1>The Notary Project</h1>
+<p><a href="https://github.com/notaryproject/notaryproject">The Notary Project</a> contains the scenario definitions for implementing container secure supply chain, which drives the Notary specification. The Notary Project is also the umbrella project for the Notary specification and Notation, its reference implementation.</p>
```
[the remark this answers is not in the document]
Comment (FeynmanZhou) [reply content/en/_index.html:25]: Agree. Not just containers. Notation can also sign artifacts.

[the remark this answers is not in the document]
Comment (FeynmanZhou) [reply content/en/_index.html:25]: +1. https://github.com/notaryproject/notaryproject is the spec and its target audience is Notation developers. 

If we want to guide new users to the GitHub project, the notation repo might be a good choice.

File: content/en/_index.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (zr-msft) [reply content/en/_index.html:19]: fixed.

File: content/en/_index.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (zr-msft) [reply content/en/_index.html:25]: removed link.

[the remark this answers is not in the document]
Comment (zr-msft) [reply content/en/_index.html:25]: fixed

File: content/en/_index.html [hunk 3]
```diff
@@ -16,27 +16,30 @@
...
+<br/>
+
+<h3>Notation</h3>
+<p>Notation is a reference implementation of the Notary specification and has three components: <a href="https://github.com/notaryproject/notation-core-go">a Go library</a> for the core cryptography of signing and verifying, <a href="https://github.com/notaryproject/notation-go">another Go library</a> that builds on the core go library and adds functionality such as pushing and pulling from registries, and <a href="https://github.com/notaryproject/notation">a CLI (<code>notation</code>)</a> that uses the go libraries to expose the full implementation.</p>
```
[the remark this answers is not in the document]
Comment (zr-msft) [reply content/en/_index.html:33]: fixed

File: content/en/_index.html [hunk 4]
```diff
@@ -16,27 +16,30 @@
...
+<p>In most cases, signing, verifying, and moving artifacts happens using <code>notation</code>.</p>
+<br/>
+
+<h2>Why use Notary and <code>notation</code>?</h2>
+<p>Notary is based on standards, such as COSE and OCI Reference Types, which provides flexibility and makes it easier to extend artifacts without impacting overall performance.</p>
```
[the remark this answers is not in the document]
Comment (zr-msft) [reply content/en/_index.html:38]: removed section

**Comment by zr-msft** [comment]
@toddysm  @FeynmanZhou  thanks for the feedback. I incorporated it. Please let me know what you think

File: content/en/_index.html [hunk 5]
```diff
@@ -16,27 +16,28 @@
...
+<br/>
+
+<h3>Notation</h3>
+<p>Notation is a reference implementation of the Notary specification and has three components: <a href="https://github.com/notaryproject/notation-core-go">a Notary Core Go library</a> for the core cryptography of signing and verifying, <a href="https://github.com/notaryproject/notation-go">Notary Go library</a> that builds on the core go library and adds functionality such as pushing and pulling from registries, and <a href="https://github.com/notaryproject/notation">a CLI (<code>notation</code>)</a> that uses the go libraries to expose the full implementation.</p>
```
Comment (yizha1) [inline content/en/_index.html:33]: Should we change `Notary Core Go library` to `Notation Core Go library`, `Notary Go library` to `Notation Go library`?

Comment (zr-msft) [reply content/en/_index.html:33]: fixed

File: content/en/_index.html [hunk 6]
```diff
@@ -16,27 +16,28 @@
...
-  </ul>
+<h1>The Notary Project</h1>
+<p>The Notary Project contains the specifications and tooling for implementing secure supply chain, which drives the Notary specification. The Notary Project is also the umbrella project for the Notary specification and Notation, its reference implementation.</p>
+<br/>
+
+<h3>Notary</h3>
+<p>Notary is a <a href="https://github.com/notaryproject/notaryproject/blob/main/signature-specification.md">specification</a> for signing and verifying container images and storing them as <a href="https://github.com/opencontainers/artifacts">OCI artifacts</a> using <a href="https://github.com/opencontainers/oci-conformance">OCI-conformant</a> registries. The latest version is Notary v2 which uses the <a href="https://www.iana.org/assignments/cose/cose.xhtml">COSE signature envelope</a> and <a href='https://github.com/opencontainers/wg-reference-types/blob/main/docs/proposals/PROPOSAL_E.md'>OCI Reference Types</a>.</p>
+<br/>
+
+<h3>Notation</h3>
```
[the remark this answers is not in the document]
Comment (zr-msft) [reply content/en/_index.html:32]: This is mentioned on line 29

**Comment by FeynmanZhou** [comment]
Close this PR as we redesigned the landing page

**Pull request closed without merging**